### DISCUSSION TITLE
Weird Error

### DISCUSSION THREAD

**Discussion by martweiss** [hbldh/bleak#1112] [Q&A]
I have some simple code, code from [here](https://bleak.readthedocs.io/en/latest/usage.html) to be specific (2nd snippet), and trying to connect an HM-10. I keep getting '_bleak_winrt_Windows_Foundation.IAsyncOperation' object has no attribute 'add_done_callback' then the program fails. What is going wrong here?

**Comment by dlech** [comment]
This has been fixed on the develop branch but hasn't been released yet.

**Reply by martweiss** [reply]
Any timeline for that release?

**Reply by dlech** [reply]
[v0.19.2](https://github.com/hbldh/bleak/releases/tag/v0.19.2) has been published with this fix.
